bin/public/installKvmHost.py

```diff
@@ -188,3 +190,11 @@ def _abort_kvm_host_installation():
 
     '''
     raise Exception("Abort kvm host installation.")
+
+def _libvirt_init_config():
+
+    x("yum install augeas -y")
+    #Initialize augeas
+    augeas = Augeas(x)
+
+    augeas.set_enhanced("/files/etc/sysconfig/libvirt-guests/ON_SHUTDOWN shutdown")
```

Comment:
The augeas path and value are different parameters and can't be specified with whitespace inside the string... set_enhanced("/files/etc/sysconfig/libvirt-guests/ON_SHUTDOWN", "shutdown")